src/Engine/Objects/Actor.h

```diff
@@ -235,7 +235,7 @@ class Actor {
     Pid lastCharacterIdToHit;
     int uniqueNameIndex = 0; // Index into pMonsterStats->pUniqueNames for a unique monster name. Regular name is used if this field is 0.
     bool donebloodsplat{ false };
-    int massDistortionTime = -1; // Value of pMiscTimer when mass distortion was cast. This was stored in the buffs table
+    Duration massDistortionTime; // Value of pMiscTimer when mass distortion was cast. This was stored in the buffs table
```

Comment:
Is it duration? It feels like point of time.

Reply:
Hmm. That's a good point, but not for the reason stated X).

`Time` in OE is like unixtime in the real world – absolute time on Enroth, measured in ticks since The Silence. This one is a time offset relative to some unspecified point in time in the past (when `pMiscTimer` was started), so it's a `Duration` and not `Time`.

However, `pMiscTimer` has nothing to do with Enroth time. It doesn't run at the same speed as Enroth time. Thus, all the `toCivilDuration` and other methods don't make sense for it. Therefore, it's actually a `std::chrono::duration` masquerading as `Duration`, since it's an animation timer measuring real time.

Again, I will leave a TODO.